Makefile

```diff
@@ -31,11 +31,8 @@ help: ## Display this help.
 
 ##@ Development
 
-# Produce CRDs that work back to Kubernetes 1.11 (no version conversion)
-CRD_OPTIONS ?= "crd:trivialVersions=true,preserveUnknownFields=false"
```

Comment:
@umangachapagain Do we still require options for these ? These flags are not available in this version

Reply:
Probably not.